libdnf/transaction/Swdb.cpp

```diff
@@ -112,12 +112,12 @@ Swdb::Swdb(const std::string &path)
                 // connect to a new database and initialize it; old data is not migrated
                 conn = std::make_shared<SQLite3>(path);
                 Transformer::createDatabase(conn);
-            } catch (std::exception & ex) {
+            } catch (SQLite3::Error & ex) {
                 // unpriviledged user may have insufficient permissions to create the database -> in-memory fallback
                 conn = std::make_shared<SQLite3>(":memory:");
                 Transformer::createDatabase(conn);
                 auto logger(libdnf::Log::getLogger());
-                logger->error(tfm::format("History database cannot be created: %s. Using in-memory database instead. Error: %s", path, ex.what()));
+                logger->error(tfm::format("History database cannot be created: %s", ex.what()));
```

Comment:
The old message:
`History database cannot be created: /opt/history.sqlite. Using in-memory database instead. Error: SQLite error on "/opt/history.sqlite": Open failed: bad parameter or other API misuse`

The new message:
`History database cannot be created: /opt/history.sqlite. SQLite error on "/opt/history.sqlite": Open failed: bad parameter or other API misuse`.

I do consider information `Using in-memory database instead.` as an essential.

Reply:
Oh, sorry, I've deleted that by accident. I'll put it back.